helm/cluster-apps-operator/values.yaml

```diff
@@ -1,4 +1,7 @@
-base: ginger.eu-west-1.aws.gigantic.io
+base: example.gigantic.io
+
+provider:
+  kind: ""
```

Comment:
Passed in from `config` here https://github.com/giantswarm/config/blob/gamma_auto_config/default/apps/cluster-apps-operator/configmap-values.yaml.template#L1-L2

Reply:
I think we can add it to the default